rt_bi_runtime:
  dd_rdf_1:
    ros__parameters:
      fuseki_server: "http://192.168.1.164:8090"
      rdf_store: "rt-bi-2"
      rdf_dir: "rdf" # Don't forget to add the resource directories to setup.py
      sparql_dir: "sparql" # Don't forget to add the resource directories to setup.py
      sparql_template_ids: "template.ids.rq"
      sparql_template_sets: "template.namespace.rq"
      sparql_var_selectors:
        - '["color", "var.color.rq"]'
        - '["name", "var.name.rq"]'
        - '["accessible-times", "var.accessibility.rq"]'
        - '["polygons", "var.polygons.rq"]'
        - '["traversability", "var.traversability.rq"]'
        - '["intervals", "var.intervals.rq"]'
      placeholder_bind: "# BIND ######################################"
      placeholder_ids: "# ID VALUES #############################"
      placeholder_order: "# ORDER VARIABLES ###########################"
      placeholder_selector: "# SELECTOR ##################################"
      placeholder_variables: "# SELECTED VARIABLES ########################"
      transition_grammar_dir: "rdf" # Don't forget to add the resource directories to setup.py
      transition_grammar_file: "transition.lark"
